tests/sentry/seer/code_review/test_webhooks.py

```diff
@@ -673,60 +684,38 @@ def setUp(self) -> None:
             "repository": {
                 "id": 12345,
                 "full_name": "owner/repo",
+                "html_url": "https://github.com/owner/repo",
             },
         }
-
-    def _enable_code_review(self) -> None:
-        self.organization.update_option("sentry:enable_pr_review_test_generation", True)
+        return orjson.dumps(event)
 
     @patch("sentry.seer.code_review.webhooks.handlers._schedule_task")
     def test_skips_when_code_review_not_enabled(self, mock_schedule: MagicMock) -> None:
-        handle_issue_comment_event(
-            event_type="issue_comment",
-            event=self.event,
-            organization=self.organization,
-            repo=self.repo,
-        )
+        event = self._build_issue_comment_event(f"Please {SENTRY_REVIEW_COMMAND} this PR")
+        self._send_issue_comment_event(event)
         mock_schedule.assert_not_called()
 
     @patch("sentry.seer.code_review.webhooks.handlers._schedule_task")
+    @with_feature({"organizations:gen-ai-features", "organizations:code-review-beta"})
     def test_skips_when_no_review_command(self, mock_schedule: MagicMock) -> None:
         self._enable_code_review()
-        event = {
-            "comment": {
-                "id": 123456789,
-                "body": "This is a regular comment without the command",
-            },
-        }
-        handle_issue_comment_event(
-            event_type="issue_comment",
-            event=event,
-            organization=self.organization,
-            repo=self.repo,
-        )
+        event = self._build_issue_comment_event("This is a regular comment without the command")
+        self._send_issue_comment_event(event)
         mock_schedule.assert_not_called()
 
     @patch("sentry.seer.code_review.webhooks.task.make_seer_request")
-    @patch("sentry.seer.code_review.webhooks.issue_comment._add_eyes_reaction_to_comment")
+    @patch("sentry.integrations.github.client.GitHubApiClient.create_comment_reaction")
     @with_feature({"organizations:gen-ai-features", "organizations:code-review-beta"})
     def test_adds_reaction_and_forwards_when_valid(
-        self, mock_reaction: MagicMock, mock_seer: MagicMock
+        self, mock_create_reaction: MagicMock, mock_seer: MagicMock
     ) -> None:
         self._enable_code_review()
-        mock_integration = MagicMock()
+        event = self._build_issue_comment_event(f"Please {SENTRY_REVIEW_COMMAND} this PR")
 
         with self.tasks():
```

Comment:
You can add `self.options({"name_of_option": False}` (I think) to make this test pass.